src/dataframeit/errors.py

```
@@ -129,11 +130,21 @@ def validate_provider_dependencies(provider: str):
 
     Raises:
         ImportError: Com mensagem amigável se dependência não estiver instalada.
     """
+    # Claude Code SDK não precisa de LangChain
+    if provider == 'claude_code':
+        try:
+            importlib.import_module('claude_agent_sdk')
+        except ImportError:
+            raise ImportError(_get_missing_package_message(
+                'claude_agent_sdk', 'claude-agent-sdk', 'Claude Code SDK'
+            ))
```

Ruff (0.15.6)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)